docs/self-managed/operate-deployment/operate-configuration.md

```diff
@@ -27,17 +27,19 @@ By default, the configuration for Operate is stored in a YAML file (`application
   - [A snippet from application.yml](#a-snippet-from-applicationyml-2)
 - [Zeebe Elasticsearch or OpenSearch exporter](#zeebe-elasticsearch-or-opensearch-exporter)
   - [Settings to connect and import](#settings-to-connect-and-import)
-  - [A snippet from application.yml for Elasticsearch:](#a-snippet-from-applicationyml-for-elasticsearch)
-  - [A snippet from application.yml for Opensearch:](#a-snippet-from-applicationyml-for-opensearch)
+  - [Snippet from application.yml for Elasticsearch](#snippet-from-applicationyml-for-elasticsearch)
+  - [Snippet from application.yml for OpenSearch](#snippet-from-applicationyml-for-opensearch)
 - [Operation executor](#operation-executor)
-  - [A snippet from application.yml](#a-snippet-from-applicationyml-3)
+  - [Snippet from application.yml](#snippet-from-applicationyml)
 - [Monitoring Operate](#monitoring-operate)
   - [Versions before 0.25.0](#versions-before-0250)
 - [Logging](#logging)
   - [JSON logging configuration](#json-logging-configuration)
   - [Change logging level at runtime](#change-logging-level-at-runtime)
     - [Set all Operate loggers to DEBUG](#set-all-operate-loggers-to-debug)
-- [An example of application.yml file](#an-example-of-applicationyml-file)
+- [Example of application.yml file](#example-of-applicationyml-file)
+
+<!--- Given we see this in the sidebar to the right of the page, can we remove this section? I feel like it makes it look crowded, and I guarantee a lot of users just skip over it. --->
```

Comment:
I've seen it before, and I think it's autogenerated from a code editor and adds a comprehensive table of contents. I agree with you, we can remove it.

I don't think it's worth an extensive review of the docs, but as we see these table of contents, we can remove them.